### ISSUE TITLE
Encourage `<-` for assignment's because its more accepted by the R Community

### ISSUE THREAD

**Issue by cansavvy** [jhudsl/intro_to_r#93]
`=` is generally not considered good R style by most style guides or by the R community. I think it's best we don't try to rock the boat on this one for the students. 

If people want more information about the difference we can refer them to this: 
https://stackoverflow.com/questions/1741820/what-are-the-differences-between-and-assignment-operators-in-r

**Comment by carriewright11** [comment]
Yeah we had discussions about this in the last class. I generally agree with this as well. A lot of the previous materials already had `=` thus this is why it is currently in the material

**Comment by cansavvy** [comment]
We should probably pick some style guide for this material to adhere to https://google.github.io/styleguide/Rguide.html or the tidyverse style guide: https://style.tidyverse.org/ or use `styler`

We could bring this up to the students, but I also understand we have to make priorities and we can't bring up style guides too early because we don't want students to be worried about too many things early on. But we should still model some of these ideas and best practices even if we don't explicitly address them in the course.

**Comment by carriewright11** [comment]
Yes, I agree - being consistent is helpful to the students and indeed explaining that they might encounter `=` as well is a good idea. 

The tidyverse style guide works for me.

**Comment by avahoffman** [comment]
Having an auto-styler sounds great. Based on last semester, I don't think they're at the level to need to know the difference between `=` and `<-`. But it would be good to tell them they might run into it.

**Comment by carriewright11** [comment]
I think we changed over all of these... we don't have an auto-styler but maybe we should make a  new enhancement issue about that? what do you think @avahoffman ?
